### ISSUE TITLE
Sound when TalkBack focus shifts to / from Braille dots; OR full-screen BrailleDotsView?

### ISSUE THREAD

**Issue by zuevval** [braille-systems/learn-braille#291]
Viktoria Nikolaevna complained that it's hard for blind users to distinguish where are the bounds of the six dots area and suggested an idea: to make a sound (a ringing bell) when the finger moves to or from dots.

I've figured out that it's possible to listen to accessibility events, e. g. focus changed; though we cannot catch the moment when the finger goes off dots: the focus shifts only when the finger reaches another element (a button, text field, etc) so it's already too late to ring (the sound will not help, what is worse, it will mix with TalkBack announcements).

I suppose we may instead add a button which will open Braille dots in full-screen (place this button in "other options" drop-down menu in theory and in the left side button stack in other modes). In the full-screen mode there will be only Braille dots, title (for example, "letter A" or "enter letter A") and a "return back" button.

**Comment by winter-yuki** [comment]
If user cannot find dots in general, he will always use "big dots mode" so drop-down menu can bee too far. But we have no other proper places. 

Maybe it willl be better to point out in the help that dots are placed in the middle of the bottom part of the screen. And maybe make left and right buttons more narrow -  they can be found by searching from the screen edges, and make dots just bigger.
